<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VidFetch</class>
 <widget class="QWidget" name="VidFetch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1264</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VidFetch v1.0  (GPLv3)</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
color: white;
background-color: rgb(60, 60, 60);
}

QLineEdit {
    border: 2px solid #757575;
    border-radius: 12px;
    padding: 0 8px;
    background: rgb(94, 92, 100); 
    color: #FFFFFF; 
    font-size: 16px;
    height: 20px;
}

QLineEdit:focus {
    border-color: rgb(107, 0, 255)
}

QLineEdit:disabled {
    background: #444444;  
    color: #aaaaaa; 
    border-color: #aaaaaa;
}

QPushButton {
        background-color: #5a2a82;
        color: #ffffff; 
        border: none;
        border-radius: 10px;
        padding: 5px 20px; 
        font-size: 12px;
        outline: none;
    }
    
    QPushButton:hover {
        background-color: #7b3ca3; 
    }

    QPushButton:pressed {
        background-color: #481f61; 
    }

    QPushButton:disabled {
        background-color: #3f1d4d; 
        color: #8a7b9a; 
    }

QProgressBar {
    border: 2px solid #5a2a82;
    border-radius: 5px;
    text-align: center;
    background-color: rgb(94, 92, 100);
}

QProgressBar::chunk {
    background-color: #5a2a82;
    width: 10px; /* You can adjust this to change the width of the 'chunk' */
}
QRadioButton {
	color: (255,255,255)}

QRadioButton::indicator::unchecked {
	border: 1px solid white;
	border-radius: 5px;
}


QRadioButton::indicator:checked {
    border : 4px solid;
	border-color: black;
	border-radius: 6px;
	background-color: rgb(0, 255, 183);

}

QSlider::groove:horizontal {
    border: 1px solid #5a2a82;
    height: 8px;
    background: #e0e0e0;
    margin: 0px;
    border-radius: 4px;
}

QSlider::handle:horizontal {
    background: #5a2a82;
    border: 1px solid #5a2a82;
    width: 18px;
    margin: -6px 0;
    border-radius: 9px;
}

QSlider::add-page:horizontal {
    background: #e0e0e0;
}

QSlider::sub-page:horizontal {
    background: #5a2a82;
}

QTreeWidget {
    background-color: rgb(94, 94, 94);
    color: white;
}

QTreeWidget::header {
    background-color: rgb(94, 94, 94);
    color: black; /* Set color to black or any other color that you prefer for the header text */
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2">
      <widget class="QPushButton" name="button_start_file">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_file">
       <property name="styleSheet">
        <string/>
       </property>
       <property name="text">
        <string>File:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_video">
       <property name="text">
        <string>Video:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineedit_video_url">
       <property name="placeholderText">
        <string>Enter YouTube video URL</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="button_start_playlist">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineedit_file">
       <property name="placeholderText">
        <string>Enter file with URLs   (Separated by white spaces)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineedit_playlist_url">
       <property name="placeholderText">
        <string>Enter YouTube Playlist URL</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_playlist">
       <property name="text">
        <string>Playlist:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="button_start_video">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeWidget" name="tree_widget">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="button_download_tree_widget">
         <property name="text">
          <string>Download selected</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="download_select_all">
         <property name="text">
          <string>Select all</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="download_unselect_all">
         <property name="text">
          <string>Unselect all</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupbox_settings">
     <property name="title">
      <string>Settings:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="1">
         <widget class="QRadioButton" name="radio_quality_ask">
          <property name="text">
           <string>Ask everytime</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QRadioButton" name="radio_quality_highest">
          <property name="text">
           <string>Highest available</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="radio_mode_music">
          <property name="text">
           <string>Music</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="radio_mode_video">
          <property name="text">
           <string>Video</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QRadioButton" name="radio_codec_aac">
          <property name="text">
           <string>AAC     (recommended for Apple devices)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QRadioButton" name="radio_codec_mp3">
          <property name="text">
           <string>MP3</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_quality">
          <property name="text">
           <string>Quality:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_output_path">
          <property name="text">
           <string>Output path:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QLineEdit" name="lineedit_output_path"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_download_mode">
          <property name="text">
           <string>Download Mode: </string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_music_codec">
          <property name="text">
           <string>Music codec</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="button_settings_save">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="1">
      <widget class="QProgressBar" name="progressbar_download">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_downloading">
       <property name="text">
        <string>Downloading:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_converting">
       <property name="text">
        <string>Converting:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QProgressBar" name="progressbar_converting">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QProgressBar" name="progressbar_processing">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_processing">
       <property name="text">
        <string>Processing:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
